builds/misc/templates/mqtt-perf-setup.yaml

```diff
@@ -42,8 +42,8 @@ jobs:
         runOptions: commands
         commands: |
           echo "$(registry.password)" | docker login $(registry.address) --username $(registry.user) --password-stdin 2>/dev/null
-          docker pull $(brokerImageName)
-          docker run -d --network=host --name=mqtt-broker $(brokerImageName)
+          docker pull $(cat brokerImageName.txt)
+          docker run -d --network=host --security-opt apparmor=unconfined --name=mqtt-broker $(cat brokerImageName.txt)
```

Comment:
I am okay with this.. but what problems are we running into with the default profile?

Reply:
Sometimes, `docker rm -f` or `docker restart` failed with permission denied issue. Googling showed that this is b/c of apparmor. I'm not sure about the details, but I have not had this issue again since I disabled apparmor.